Skyra/Database/Models/User.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace Skyra.Database.Models
+{
+	public class User
+	{
+		public ulong Id { get; set; }
+		public uint CommandUses { get; set; } = 0;
+		public string[] BannerList { get; set; } = new string[0];
+		public string[] BadgeSet { get; set; } = new string[0];
+		public string[] BadgeList { get; set; } = new string[0];
+		public uint Color { get; set; } = 0x000000;
+		public ulong[] Marry { get; set; } = new ulong[0];
+		public long Money { get; set; } = 0;
+		public uint PointCount { get; set; } = 0;
+		public uint ReputationCount { get; set; } = 0;
+		public string ThemeLevel { get; set; } = "1001";
+		public string ThemeProfile { get; set; } = "0001";
+		public bool DarkTheme { get; set; } = false;
+		public bool ModerationDm { get; set; } = true;
+		public DateTime? NextDaily { get; set; } = null;
+		public DateTime? NextReputation { get; set; } = null;
+		public int Vault { get; set; } = 0;
```

Comment:
can the Vault be negative? If not, consider using an unsigned type - perhaps ushort?

Reply:
Actually that should be `ulong`